app/views/chatrooms/index.html.slim

```diff
@@ -0,0 +1,38 @@
+.container.mt-3
+  h2
+    = t ".title-edit"
+  p
+    = t ".mess"
+  - if current_user.admin?
+    form.button_to[method="get"action="chatrooms/new"]
+      input.btn.btn-success[value="New Chatrooms" type="submit"]
+  input#myInput.form-control[type="text" placeholder="Search.."]
+  br
+  table.table.table-striped.table-condensed
+    thead
+      tr
+        th
+          = t ".lb-channel"
+        th
+           = t ".lb-date"
+        th
+           = t ".lb-number"
+        th
+          = t ".lb-action"
+    tbody#myTable
+      - @chatrooms.each do |chatroom|
+        tr
+          td = chatroom.name
+          td = chatroom.created_at
+          td
+            | 10 thang
+          td
+            .space
+              -if current_user.admin?
+                = link_to  = t(".btn-destroy"), chatroom, method: :delete, data: { confirm:  = t(".mess-sure") }
```

Comment:
what the hell syntax `= link_to  = t(".btn-destroy")` ?

Reply:
có một sự rối nhẹ :))
fixed